app/controllers/users.js

```diff
@@ -10,3 +11,8 @@ exports.createUser = (req, res, next) => {
     .then(user => res.status(201).send(showUserSerializer(user)))
     .catch(next);
 };
+
+exports.usersIndex = (req, res, next) =>
```

Comment:
I think getUsers is a more appropriate name, but it's ok :+1:

Reply:
:rails: